### PR TITLE
Updated homepage twig, regions and theming

### PR DESCRIPTION
**Pull request by KMDOFNI**
Updated homepage twig, regions and theming

### CODE REVIEW

File: project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/components.scss [hunk 1]
```diff
@@ -24,4 +24,39 @@
...
+  font-weight: normal;
+}
+
+
+#main-menu .menu-link, #main-menu .menu {
+  background-color: #fff;
+  color: $brand-primary;
+}
+
+#main-menu .menu .menu-item .active, #main-menu .menu .menu-item .active + .menu-main_sub .title a {
```
Comment (soda736) [inline project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/components.scss:55]: When targeting 2 selectors you should put the 2nd/3rd etc. on new lines for better readability.
You should also always use hex values or variables for colours. "blueviolet" should be a hex e.g. #878787

File: project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/components.scss [hunk 2]
```diff
@@ -24,4 +24,39 @@
...
+  display: flex;
+  text-align: left;
+  padding: 4rem 0 5rem;
+  border-bottom: 1px solid $brand-secondary;
+
+  .left {
+    flex: 2;
+  }
+
+  .right{
```
Comment (soda736) [inline project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/components.scss:37]: Should have a space between right and the opening bracket.

**Review by soda736 — CHANGES_REQUESTED** [review]

File: project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/components.scss (hunk 1 above)
Comment (KMDOFNI) [reply project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/components.scss]: Thanks Adam , I have now fixed this

File: project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/components.scss (hunk 2 above)
Comment (KMDOFNI) [reply project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/components.scss]: Thanks Adam , I have now fixed this

**Pull request merged**